### DISCUSSION TITLE
p/975b0b1dcda549de99e488a844f3559e/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#37] [Announcements]
# p/975b0b1dcda549de99e488a844f3559e/

散策今週はあまり出かけなくて、月曜日と水曜日しか出かけない。別々に腰小山と金閣寺へ行った。 月曜日月曜日に小塩山の辺りに行った。先ずは山の下の大原野神社へ行って、隣は正法寺があるけど、有料なので入らなかった。そらから、小塩山に登った。本来なら王城の滝に興味があるけど、見つからなかったり、それに時間が遅いしので、それで早く山を降りた。 神社の正門  神社の庭園  夜に山を降りた  隣の有料な正法寺

https://blog.rxliuli.com/p/975b0b1dcda549de99e488a844f3559e/

**Comment by miaopass-future** [comment]
只用多邻国不背单词句子就能学到写小作文的程度么

**Reply by rxliuli** [reply]
当然不是，吾辈使用 anki 背单词，不过新一似乎是只用 duolingo 的
